examples/sample-pub/publisher.js

```diff
@@ -144,6 +144,36 @@ function startFlowAuto() {
     return;
   }
 
+  if (flow == 'metering') {
+    whenReady((subsciptions) => {
+      subsciptions
+        .getEntitlements({
+          metering: {
+            userState: {
+              id:
+                'user5901e3f7a7fc5767b6acbbbaa927d36f5901e3f7a7fc5767b6acbbbaa927',
+              attributes: [
+                {
+                  name: 'registered_user',
+                  timestamp: (Date.now() / 1000) | 0,
+                },
+              ],
+            },
+          },
+        })
+        .then((entitlements) => {
+          if (entitlements.enablesThisWithGoogleMetering()) {
+            console.log('Hey Google, thanks for the metering entitlement.');
+          } else {
+            console.log(
+              'I was promised metering entitlements... (╯°□°)╯︵ ┻━┻'
```

Comment:
Change to "Sorry no meter entitlements today"

Reply:
The message has been updated